extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -140,8 +140,64 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		await this.bookTocManager.createBook(bookPath, contentPath);
 	}
 
-	async editBook(book: BookTreeItem, section: BookTreeItem): Promise<void> {
-		await this.bookTocManager.updateBook(section, book);
+	async editBook(movingElement: BookTreeItem): Promise<void> {
+		try {
+			let bookOptions: vscode.QuickPickItem[] = [];
+			let pickedSection: vscode.QuickPickItem;
+			let updateBook: BookTreeItem;
+			let bookSections: JupyterBookSection[];
+			this.books.forEach(book => {
+				if (!book.isNotebook) {
+					bookOptions.push({ label: book.bookItems[0].title, detail: book.bookPath });
+				}
+			});
+			let pickedBook = await vscode.window.showQuickPick(bookOptions, {
+				canPickMany: false,
+				placeHolder: loc.labelBookFolder
+			});
+
+			if (pickedBook && movingElement) {
+				updateBook = this.books.filter(book => book.bookPath === pickedBook.detail)[0].bookItems[0];
+				if (updateBook) {
+					bookSections = updateBook.sections;
+					while (bookSections?.length > 0) {
+						bookOptions = [{ label: loc.labelAddToLevel, detail: pickedSection ? pickedSection.detail : '' }];
+						bookSections.forEach(section => {
+							if (section.sections) {
+								bookOptions.push({ label: section.title, detail: section.file });
+							}
+						});
+						bookSections = [];
+						if (bookOptions.length > 1) {
+							pickedSection = await vscode.window.showQuickPick(bookOptions, {
+								canPickMany: false,
+								placeHolder: loc.labelBookSection
+							});
+							if (pickedSection.label && pickedSection.label === loc.labelAddToLevel) {
+								break;
+							}
+							else if (pickedSection.detail) {
+								bookSections = updateBook.findChildSection(pickedSection.detail).sections;
+							}
+						}
+					}
+					if (pickedSection) {
+						const targetSection = pickedSection.detail !== undefined ? updateBook.findChildSection(pickedSection.detail) : undefined;
+						if (movingElement.tableOfContents.sections) {
+							// this is for notebooks what about sections
+							if (movingElement.contextValue === 'savedNotebook') {
```

Comment:
are you saying there is still to-do work for sections? can you clarify?

Reply:
The work of sections is there as well. I forgot to remove this comment